lib/iris/cube.py

```diff
@@ -1587,28 +1577,7 @@ def dtype(self):
         :class:`~iris.cube.Cube`.
 
         """
-        return self._data_manager.dtype
-
-    @property
-    def fill_value(self):
-        """
-        A fill value for the data of the :class:`~iris.cube.Cube`.
-
-        This is a value suitable for filling masked points in this cube's data.
-        It may be ``None``, meaning that no suitable fill value is known.
-
-        .. Note::
-
-            Ideally, this value will not occur anywhere in the cube data.
-            Thus, many operations that change cube data will also set
-            ``fill_value`` to ``None``.
-
-        """
-        return self._data_manager.fill_value
-
-    @fill_value.setter
-    def fill_value(self, fill_value):
-        self._data_manager.fill_value = fill_value
+        return self._data_manager.core_data().dtype
```

Comment:
@lbdreyer Isn't this just `self._data_manager.dtype` still? Why the need to go through the `core_data` function?